readme.md

```diff
@@ -1,6 +1,7 @@
 # SteamKit2
 
-[![Build Status](https://img.shields.io/travis/SteamRE/SteamKit.svg?style=flat-square)](https://travis-ci.org/SteamRE/SteamKit)
+[![Build Status (Mono)](https://img.shields.io/travis/SteamRE/SteamKit.svg?style=flat-square)](https://travis-ci.org/SteamRE/SteamKit)
+[![Build Status (Windows)](https://ci.appveyor.com/api/projects/status/tk28c1nnjql3pgr9?svg=true)](https://ci.appveyor.com/project/SteamRE/SteamKit)

```

Comment:
Perhaps it would make sense to use shields.io for this with a custom label: `https://img.shields.io/appveyor/ci/SteamRE/SteamKit.svg?style=flat-square&label=Windows`

![](https://img.shields.io/appveyor/ci/SteamRE/SteamKit.svg?style=flat-square&label=Windows)